apps/framework-docs/src/pages/on-prem/deploying-on-kubernetes.mdx

```diff
@@ -98,7 +98,7 @@ This will allow Kubernetes to determine the health of your moose service.
 
 ## Configuring container environment variables
 
-    Inside your `moose-deployment.yaml` file you'll need to add an `env` section for environment variables. In the following example snippit we're using Kubernetes secretes which is an industry best practice.
+    Inside your `moose-deployment.yaml` file you'll need to add an `env` section for environment variables. In the following example we're using Kubernetes secretes which is an industry best practice.
```

Comment:
Minor typo in "secretes"

Reply:
Not minor! Darn I meant to get that one! Will update this PR. Thanks @DatGuyJonathan